### PR TITLE
Implement tracing policy

### PR DESCRIPTION
**Pull request by vietj**

### CODE REVIEW

File: vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingTracer.java [hunk 1]
```diff
@@ -56,28 +58,37 @@ public OpenTracingTracer(boolean closeTracer, Tracer tracer) {
...
 
-      @Override
-      public void put(String key, String value) {
-        throw new UnsupportedOperationException();
+        @Override
+        public void put(String key, String value) {
+          throw new UnsupportedOperationException();
+        }
+      });
+      if (sc != null || policy == TracingPolicy.SUPPORT) {
```
Comment (jotak) [inline vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingTracer.java:79]: "SUPPORT" means "ALWAYS" ? I don't really get why you chose that word, "always" seemed more clear to me

**Review by jotak — APPROVED** [review]
lgtm

File: vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingTracer.java (hunk 1 above)
Comment (vietj) [reply vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingTracer.java:79]: it should be required here :-)

Comment (vietj) [reply vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingTracer.java:79]: but it will be always I guess

**Pull request merged**